dns/cloudflare.go

```diff
@@ -148,6 +156,7 @@ func (cf *Cloudflare) create(zoneID string, domain *config.Domain, recordType st
 		TTL:     cf.TTL,
 	}
 	record.Proxied = domain.GetCustomParams().Get("proxied") == "true"
+	record.Comment = domain.GetCustomParams().Get("comment")
```

Comment:
```suggestion
		TTL:     cf.TTL,
		Comment: domain.GetCustomParams().Get("comment"),
	}
	record.Proxied = domain.GetCustomParams().Get("proxied") == "true"
```

只是好奇，是不是只用在创建记录时添加 `comment` 参数就够了？

Reply:
感谢建议，但我不是很理解您的问题。查询的时候需要comment参数来筛选记录，只有comment相同的记录会被更新